src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_NamingStyle.cs

```diff
@@ -102,5 +103,29 @@ private static bool TryParseCapitalizationScheme(string namingStyleCapitalizatio
                     return false;
             }
         }
+
+        public static string ToEditorConfigString(this Capitalization capitalization)
+        {
+            switch (capitalization)
+            {
+                case Capitalization.PascalCase:
+                    return "pascal_case";
+
+                case Capitalization.CamelCase:
+                    return "camel_case";
+
+                case Capitalization.FirstUpper:
+                    return "first_word_upper";
```

Comment:
nit: it woudlbe nice if the enum value matched this text.

Reply:
➡️ These were already defined

Reply:
Understood.  Would still like them to match :D  (i'm not pushing on it, hence hte 'nit'), just something i noticed :)